tests/unit/test_runner.py

```diff
@@ -52,41 +50,37 @@ def test_request_is_ignored_because_private(
         execute_action(
             request=webhook_create_example,
             actions=actions_example,
-            settings=settings,
         )
 
     assert str(exc_info.value) == "private bugs are not supported"
 
 
-def test_request_matched_whiteboard_with_dash(
-    caplog,
-    webhook_create_example,
-    action_factory,
-    settings: Settings,
-    mocked_bugzilla,
-):
-    action_tag = "data-quality"
-    actions_example_with_inner_match = Actions.parse_obj(
-        [action_factory(whiteboard_tag=action_tag)]
-    )
-    bug = bug_factory(whiteboard=f"[{action_tag}-backlog]")
-    webhook_create_example.bug = bug
-    mocked_bugzilla.get_bug.return_value = bug
-
-    result = execute_action(
-        request=webhook_create_example,
-        actions=actions_example_with_inner_match,
-        settings=settings,
-    )
-
-    result_bug = BugzillaBug.parse_raw(result["bug"])
-    assert result_bug.id == bug.id
+# TODO:
+# def test_request_matched_whiteboard_with_dash(
+#     webhook_create_example,
+#     action_factory,
+#     mocked_bugzilla,
+# ):
+#     action_tag = "data-quality"
+#     actions_example_with_inner_match = Actions.parse_obj(
+#         [action_factory(whiteboard_tag=action_tag)]
+#     )
+#     bug = bug_factory(whiteboard=f"[{action_tag}-backlog]")
+#     webhook_create_example.bug = bug
+#     mocked_bugzilla.get_bug.return_value = bug
+
+#     result = execute_action(
+#         request=webhook_create_example,
+#         actions=actions_example_with_inner_match,
+#     )
+#     breakpoint()
+#     result_bug = BugzillaBug.parse_raw(result["bug"])
+#     assert result_bug.id == bug.id
```

Comment:
What shall we do with this?

Reply:
I decided to delete this test because whiteboard tag parsing is pretty well-covered over in `test_bugzilla.py`